src/LeanCode.ContractsGenerator/Program.cs

```diff
@@ -66,44 +66,44 @@ private static async Task<int> Main(string[] args)
         }
         catch (InvalidProjectException ex)
         {
-            Console.Error.WriteLine($"Cannot load one of the projects: {ex.Message}");
-            Console.Error.WriteLine("At");
-            Console.Error.WriteLine(ex.StackTrace);
+            await Console.Error.WriteLineAsync($"Cannot load one of the projects: {ex.Message}");
```

Comment:
TBH, that's unnecessary (although correct) noise introduced by the too-generic analyzers (they don't mark `Console.WriteLine` as being wrong, don't they).

Reply:
I don't really know - I don't like this, but I also don't like additional `SuppressMessage`... :/